src/serial_terminal.rs

```diff
@@ -312,6 +312,7 @@ fn handle_property_write(
         return;
     }
 
+    context.settings.properties = new_props;
```

Comment:
This fixes a bug in the last PR where the new settings weren't getting persisted properly.